### ISSUE TITLE
Nodejs Yarn version collision

### ISSUE THREAD

**Issue by szepeviktor** [widoz/wp-entities-search#39]
The Yarn lock file seems to be Yarn2, but Yarn2 needs Nodejs 18.

> Usage Error: This tool requires a Node version compatible with >=18.12.0 (got 16.20.2).

💥 What to do now?

**Comment by szepeviktor** [comment]
v1 lock files start with
```js
# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
# yarn lockfile v1
```

**Comment by widoz** [comment]
Hi @szepeviktor,

Thank you for pointing it out, I probably made a mixup with the local yarn version I'm using. I think make sense to update node js and I was wondering to use yarn 2 also. 

Therefore I would go with the LTS version of node. Any other opinion or suggestion?

**Comment by szepeviktor** [comment]
> Any other opinion or suggestion?

No.

**Comment by szepeviktor** [comment]
Installation with Yarn stable (v4) fails.
https://github.com/szepeviktor/that-entities-search/pull/2/files#diff-a88b0a4f767bead41b7923be71fc24ca488e65a46d253e0641842df191d36a52R25-R28

**Comment by widoz** [comment]
I think also make sense to add the `paths` as you did to the workflow file but I would do it in a different PR just to keep things clean.

**Comment by widoz** [comment]
Fixed by #41
